src/WebAdmin.cpp

```diff
@@ -340,8 +340,8 @@ void cWebAdmin::HandleWebadminRequest(cHTTPServerConnection & a_Connection, cHTT
 	// Try to get the template from the Lua template script
 	if (ShouldWrapInTemplate)
 	{
-		cCSLock Lock(m_CS);
-		cLuaState::cLock lock(m_TemplateScript);
+		cCSLock lockSelf(m_CS);
+		cLuaState::cLock lockTemplate(m_TemplateScript);
```

Comment:
I'm more inclined for javaCase, the thing I'll wage war against is silly_unreadable_t snake_case.